docs/docs/reference/functions/middleware-and-helpers.md

```diff
@@ -51,7 +51,68 @@ export default async function corsHandler(req, res) {
 
 ## Custom body parsing
 
-Generally useful for file upload support.
+Generally useful for file upload support or raw bodies.
+
+Each function can export an object named `config`. This object will allow you to control the `body-parser` middleware used by Gatsby Functions. The `limit` property will allow configuration of payload size up to **_32mb_**.
+
+```js
+ // Config object
+ {
+   bodyParser: {
+     json: {},
+     raw: {},
+     text: {},
+     urlencoded: {}
+   }
+ }
+```
+
+The properties of bodyParser currently allow for two configurable options: `limit` and `type`, which are documented by
```

Comment:
A super nitpick... What do you think of this? It might be a little confusing to go to those docs and see all the other options. Making it clear that we only support the two right now is key.

```suggestion
Gatsby currently supports the `limit` and `type` options for the `bodyParser` configuration, which are documented by
```